samples/excel/01-basics/basic-api-call-es5.yaml

```diff
@@ -34,33 +34,60 @@ script:
     language: typescript
 template:
     content: |-
-        <section class="ms-Fabric ms-font-m">
-            <p class="ms-font-m">This sample demonstrates basic Excel API calls.</p>
+        <section>
+            <p>This sample demonstrates basic Excel API calls using ES5.</p>
         </section>
-        <section class="ms-Fabric samples ms-font-m">
+        <section>
             <h3>Try it out</h3>
-            <p class="ms-font-m">Select some cells in the worksheet, then press <b>Highlight selected range</b>.</p>
-            <button id="run" class="ms-Button">
-                <span class="ms-Button-label">Highlight selected range</span>
-            </button>
+            <p>Select some cells in the worksheet, then press <b>Highlight selected range</b>.</p>
+            <button id="run">Highlight selected range</button>
         </section>
     language: html
 style:
     content: |-
-        section.samples {
-            margin-top: 20px;
+        body {
+            font-family: 'Segoe UI', Tahoma, Geneva, Verdana, sans-serif;
+            font-size: 14px;
+            line-height: 1.5;
+            padding: 10px;
         }
 
-        section.samples .ms-Button, section.setup .ms-Button {
-            display: block;
-            margin-bottom: 5px;
+        section {
+            margin-bottom: 20px;
+        }
+
+        h3 {
+            margin-top: 0;
+            margin-bottom: 10px;
+            font-size: 16px;
+        }
+
+        p {
+            margin: 0 0 10px 0;
+        }
+
+        button {
+            background-color: #0078d4;
```

Comment:
For visual accessibility, I think we should make all of these background blues (button, hover, and active states) darker. The contrast between background color and text here barely passes [WCAG AA requirements, and fails WCAG AAA](https://webaim.org/resources/contrastchecker/?fcolor=FFFFFF&bcolor=0078D4). We might as well meet WCAG AAA, unless there's a strong design reason not to.

Reply:
I suggested a darker blue here, but after further review (see samples/excel/10-chart/chart-create-several-charts.yaml) I now think black text/gray background would be more accessible.